src/tasks/installers/helm.ts

```diff
@@ -96,9 +142,13 @@ export class HelmTasks {
       },
       {
         title: 'Create Tiller RBAC',
+        // Tiller is not used anymore in helm v3
+        enabled: (ctx: any) => !ctx.isHelmV3,
```

Comment:
It may be easier to read and understand if v3 and !v3 tasks will be in different methods.

I mean something like:
```
[{check version},
 { 
  if v3 
  getV3Tasks
 else 
  get!V3Tasks
}]
```

Reply:
If I do this it will do subtree in the output

Reply:
Ok, then I'm ok to merge it as is